lib/author.rb

```diff
@@ -0,0 +1,19 @@
+class Author
+  attr_accessor :first_name, :last_name
+
+  def initialize(first_name: nill, last_name: nill)
+    @id = Random.rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.author = self
+  end
+
+  private
+
+  attr_accessor :id, :items
```

Comment:
This is not correct, I believe you are trying to put as private id and items, in order to accomplish that you don't need to do anything. by default if you don't declare your variable with `attr_accessor :` your variables are going to be private, in other words you use attr_accessor for make variables public.